services/api/src/api/config.py

```diff
@@ -88,6 +88,7 @@ class EndpointConfig:
                 "split": "split-is-valid",
             },
             "/statistics": {"split": "split-descriptive-statistics"},
+            "/compatible-libraries": {"dataset": "dataset-compatible-libraries"},
```

Comment:
why do we need this endpoint?

Reply:
For the front end to access the list of compatible libraries and the code snippets to show to the user.

Let me know if this endpoint is fine is if you have something else in mind

Reply:
Hmmm, OK, indeed, a lot of the data are not passed through sse-api (from `dataset-hub-cache`).